sdk/include/opentelemetry/sdk/metrics/aggregation/histogram_aggregation.h

```diff
@@ -19,21 +19,22 @@ class LongHistogramAggregation : public Aggregation
 public:
   LongHistogramAggregation();
   LongHistogramAggregation(HistogramPointData &&);
+  LongHistogramAggregation(const HistogramPointData &);
 
   void Aggregate(long value, const PointAttributes &attributes = {}) noexcept override;
 
   void Aggregate(double value, const PointAttributes &attributes = {}) noexcept override {}
 
   /* Returns the result of merge of the existing aggregation with delta aggregation with same
    * boundaries */
-  virtual std::unique_ptr<Aggregation> Merge(const Aggregation &delta) const noexcept override;
```

Comment:
Why is virtual removed?

Reply:
It was not required, as we won't need to subclass `HistogramAggregation` and other aggregations further.